pkgs/development/libraries/xdg-desktop-portal-gtk/default.nix

```diff
@@ -1,15 +1,15 @@
 { stdenv, fetchFromGitHub, autoreconfHook, pkgconfig, libxml2, xdg-desktop-portal, gtk3, glib }:
 
 let
-  version = "1.0.2";
+  version = "1.1.0";
 in stdenv.mkDerivation rec {
```

Comment:
The lockdown will require `gsettings-desktop-schemas`.

https://github.com/flatpak/xdg-desktop-portal-gtk/blob/14c62ab18b49e67324b9a75e3627b075c5a6067d/src/lockdown.c#L48

For an example how to do it see https://github.com/NixOS/nixpkgs/pull/42177/files#diff-81007e7b56315f92406c1cf3d7ee1d47R16

Reply:
Is tackling the lockdown aspect a blocker? I can look into a solution similar to what you've linked, but might take a bit (may need to start testing using a proper gnome session so I can access these things?).

On the other hand if the "fail" mode here isn't really "fail-safe" (such that user requests permission be denied but it's ignored since flatpak can't find the setting or something) that's no good and certainly should be addressed!

FWIW upstream has more changes re:lockdown (https://github.com/flatpak/xdg-desktop-portal/pull/267), not sure if that makes integration easier or not :).

Reply:
If schemas are not available, the program will crash.